### DISCUSSION TITLE
How to use Anilist watch list and reading list

### DISCUSSION THREAD

**Discussion by ToulthG** [lowlighter/metrics#377] [Q&A] [answered]
I don't know how to achieve my own.....

**Comment by Zenoo** [comment]
Did you read the [README](https://github.com/lowlighter/metrics/blob/master/source/plugins/anilist/README.md) for this plugin ?

**Reply by ToulthG** [reply]
> 您是否阅读了此插件的[自述文件](https://github.com/lowlighter/metrics/blob/master/source/plugins/anilist/README.md)？

But i don't know where to get the svg of anilist(I don't know how to use this plugin)

**Reply by Zenoo** [reply]
There's a guide on the main README: 

[Using GitHub Action on your profile repository (~5 min setup)](https://github.com/lowlighter/metrics#%EF%B8%8F-using-github-action-on-your-profile-repository-5-min-setup)  

The mains steps are:
- Setting up the access token
- Adding the `.github/workflow/metrics.yml` file to your profile repository

**Reply by ToulthG** [reply]
Thank you very much for your help
I will take time to understand

**Comment by spenserblack** [comment]
You can also create a preview and copy the `.yml` [here](https://metrics.lecoq.io/) to help you get started.

**Reply by ToulthG** [reply]
I will try[](https://metrics.lecoq.io/)

**Reply by spenserblack** [reply]
Please note I found a bug with the anilist config preview (#378)

**Reply by ToulthG** [reply]
Thanks for reminding，I just did this step
![image](https://user-images.githubusercontent.com/61505961/121533919-53065980-ca33-11eb-9e3e-51ba4090fd8c.png)

**Reply by spenserblack** [reply]
Is that the actual token hash?
:fearful: I think the `token` should be secret

**Reply by ToulthG** [reply]
I know the mark should be kept secret, but I have figured out a mark (no one cares about my mark) 笑

**Comment by ToulthG** [comment, marked as answer]
I made it!

**Comment by ToulthG** [comment]
[here](https://github.com/ToulthG)
